include/lbann/utils/dnn_lib/miopen/local_response_normalization.hpp

```diff
@@ -43,6 +44,7 @@ using namespace miopen;
 
 inline size_t get_lrn_ws_size(TensorDescriptor const& yDesc)
 {
+  BASIC_PROF_REGION("miopen:get_lrn_ws_size");
   size_t size;
   CHECK_MIOPEN(miopenLRNGetWorkSpaceSize(yDesc,
```

Comment:
Same thought here.  Are these calls performance killers.

Reply:
These aren't really causing concern in the profiles. Alexnet is the only model committed to LBANN that uses this. The cuDNN codepath returns `0UL`, but MIOpen will fail in this case.